faster_sam/routing.py

```diff
@@ -22,6 +35,20 @@ def __init__(self, data: Dict[str, Any]):
 
 
 async def event_builder(request: Request) -> Dict[str, Any]:
+    """
+    Builds an event dictionary from the given request.
```

Comment:
Acho que podemos complementar um pouco mais a descrição para deixar claro de qual evento estamos falando.

_Builds an event of type aws_proxy from API Gateway._

_It uses the given request object to fill the event details._